gazebo_grasp_plugin_ros/CMakeLists.txt

```diff
@@ -0,0 +1,96 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(gazebo_grasp_plugin_ros)
+
+## Compile as C++11, supported in ROS Kinetic and newer
```

Comment:
We can remove most commented stuff here;  or I can do once merged

Reply:
I removed some, but opted to keep some (for the commands which are actually done, as documentation)